airflow/configuration.py

```diff
@@ -159,6 +159,7 @@ class AirflowConfigParser(ConfigParser):  # pylint: disable=too-many-ancestors
         ('metrics', 'statsd_datadog_tags'): ('scheduler', 'statsd_datadog_tags', '2.0.0'),
         ('metrics', 'statsd_custom_client_path'): ('scheduler', 'statsd_custom_client_path', '2.0.0'),
         ('scheduler', 'parsing_processes'): ('scheduler', 'max_threads', '1.10.14'),
+        ('operators', 'default_queue'): ('celery', 'default_queue', '2.1.0'),
```

Comment:
I'm not sure what version should be put here, guessing this would have to go as a new minor?

Reply:
I have a feeling that it doesn't matter much now, because the release manager decides whether this change will be included in the next release.

Reply:
Please put 2.1 though, yes. Saves us having to change it later.